start-browser.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/zsh
+
+# cargo run
+cargo build
+(
+	sleep 1
```

Comment:
This expression is OK temporarily, but we need to check target app is running by some method (for example, the `ps` command).